app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    list_input = cp.split()
+    command, file_in, file_out = list_input
```

Comment:
Unpacking `list_input` before checking if it contains exactly three elements can lead to a `ValueError` if the input is invalid. Consider checking the length of `list_input` before unpacking.